ko/admin/admin_utils.rst

```diff
@@ -3664,6 +3665,54 @@ flashback
 
         cubrid flashback --oldest demodb dba.tbl
 
+.. _memmon:
+
+memmon
+---------
+
+ **cubrid memmon** 유틸리티는 서버 프로세스의 힙 메모리 사용량에 대한 모니터링 현황을 출력한다. **cubrid memmon** 을 수행하기 위해서는 현황 출력의 대상이 되는 서버 프로세스의 시스템 파라미터 **enable_memory_monitoring** 을 반드시 yes로 설정해야 한다. ::
+
+    cubrid memmon [option] database_name
+
+*   **cubrid**: CUBRID 서비스 및 데이터베이스 관리를 위한 통합 유틸리티
+
+*   **memmon**: 서버의 힙 메모리 사용량에 대한 모니터링 현황을 출력하는 도구
+
+*   *database_name*: 현황을 출력하고자 하는 서버 프로세스가 운영중인 데이터베이스 이름
+
+다음 예제에서는 사용자가 "cubrid memmon demodb"를 수행했을 때 출력되는 결과를 보여준다.
+
+::
+
+    ====================cubrid memmon====================
+    Server Name: demodb
+    Total Memory Usage: 935240 KB (for meta info: 470 KB)
+    -----------------------------------------------------
+            File Name                                                                                           |     Memory Usage(Ratio)
+            storage/page_buffer.c:4977                                                                          |           525315 KB( 56%)
+            transaction/log_page_buffer.c:584                                                                   |           262147 KB( 28%)
+            ...
+
+위의 예에서 **cubrid memmon** 을 실행하면 서버 프로세스가 운영중인 데이터베이스 이름, 서버 프로세스가 사용 중인 힙 메모리의 총량, 모니터링을 위한 힙 메모리 할당의 메타 정보를 저장하기 위해 사용 중인 힙 메모리의 양, 할당이 발생한 모든 파일:라인 정보 및 현재 메모리 사용량, 그리고 서버 프로세스의 전체 힙 메모리 사용량 대비 점유율을 표시한다.
+
+**cubrid memmon** 에 표시된 각 칼럼의 의미는 다음과 같다.
```

Comment:
```suggestion
유틸리티가 출력하는 정보는 다음과 같은 의미를 갖는다.
```
Please list up and describe
- Server Name
- Total Memory Usage
  - about this
  - about `or meta info`

Reply:
Done.